operator/scripts/create-olm-bundle.sh

```diff
@@ -67,7 +67,6 @@ yq ea -i '.spec.apiservicedefinitions = {}' "$CSV_PATH"
 yq ea -i '.spec.install.spec.deployments[0].spec.selector.matchLabels.name = "keycloak-operator"' "$CSV_PATH"
 yq ea -i '.spec.install.spec.deployments[0].spec.template.metadata.labels.name = "keycloak-operator"' "$CSV_PATH"
 
-yq ea -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].env += [{"name": "QUARKUS_OPERATOR_SDK_NAMESPACES", "valueFrom": {"fieldRef": {"fieldPath": "metadata.annotations['"'"'olm.targetNamespaces'"'"']"}}}]' "$CSV_PATH"
```

Comment:
The new SDK version adds the namespaces env vars for each controller automatically. It now looks like:
```
...
              spec:
                containers:
                  - env:
                      - name: QUARKUS_OPERATOR_SDK_CONTROLLERS_WATCHEDSECRETSCONTROLLER_NAMESPACES
                        valueFrom:
                          fieldRef:
                            fieldPath: "metadata.annotations['olm.targetNamespaces']"
                      - name: QUARKUS_OPERATOR_SDK_CONTROLLERS_KEYCLOAKREALMIMPORTCONTROLLER_NAMESPACES
                        valueFrom:
                          fieldRef:
                            fieldPath: "metadata.annotations['olm.targetNamespaces']"
                      - name: QUARKUS_OPERATOR_SDK_CONTROLLERS_KEYCLOAKCONTROLLER_NAMESPACES
                        valueFrom:
                          fieldRef:
                            fieldPath: "metadata.annotations['olm.targetNamespaces']"
...
```
CC @shawkins